sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -552,6 +552,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_lnl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-lnl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = BIT(0),
+		.links = sdw_mockup_multi_func,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-lnl-rt722-l0.tplg",
```

Comment:
why rt722 if this is mockup?

Reply:
We reuse the existing tplg for mockup configs.

Reply:
Can we add a comment here please?

Reply:
done